src/main/java/frc/robot/commands/Wrist/RotLeft90Cmd.java

```diff
@@ -1,52 +0,0 @@
-// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
If there's a chance we will have a rotating wrist on this robot, these files should stay, we can just remove their use in robotcontainer for now.